BlueShift-iOS-SDK/BlueShift.m

```diff
@@ -151,7 +151,7 @@ - (void) setupWithConfiguration:(BlueShiftConfig *)config {
         }
         
         if ([[BlueShiftAppData currentAppData] getCurrentInAppNotificationStatus] == YES && config.inAppManualTriggerEnabled == NO) {
-            _inAppNotificationMananger.inAppNotificationTimeInterval = [NSNumber numberWithDouble:config.BlueshiftInAppNotificationTimeInterval];
+            _inAppNotificationMananger.inAppNotificationTimeInterval = config.BlueshiftInAppNotificationTimeInterval;
```

Comment:
there is no need to check the `.doubleValue` here, right?

Reply:
yes